src/Columns.tsx

```diff
@@ -4,7 +4,7 @@ import { Column } from './common/types';
 
 // TODO: fix type
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const SelectColumn: Column<any> = {
+export const SelectColumn: Column<any, never> = {
```

Comment:
```suggestion
export const SelectColumn: Column<any> = {
```
We default it to `never` so we can skip it.